content/en/docs/writing-policies/validate.md

```diff
@@ -1889,4 +1889,8 @@ The deployments "nginx" is invalid:  ValidatingAdmissionPolicy 'disallow-host-pa
 Since Kubernetes ValidatingAdmissionPolicies are cluster-scoped resources, ClusterPolicies can only be used to generate them.
 {{% /alert %}}
 
+{{% alert title="Warning" color="warning" %}}
+When Kyverno policies match 'pods', the generated ValidatingAdmissionPolicy will match both 'pods' and 'pods/ephemeralcontainers'. This occurs because Kyverno inherently includes 'pods/ephemeralcontainers' by default in the corresponding validatingwebhookconfiguration, and we require analogous behavior for the ValidatingAdmissionPolicies.
```

Comment:
```suggestion
When a Kyverno policy matches solely on Pods, the generated ValidatingAdmissionPolicy will match both `pods` and `pods/ephemeralcontainers`. This occurs because Kyverno inherently includes `pods/ephemeralcontainers` by default in the corresponding ValidatingWebhookConfiguration, and we require analogous behavior for the ValidatingAdmissionPolicies.
```